<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lInformation">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Resolveur d'Equation du second degr√©e&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSaisirA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisir A :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lSaisirB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisir B :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lSaisirC">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisir C :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEC">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lX1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>x1 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="lX2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>x2 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="lDelta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta =</string>
    </property>
   </widget>
   <widget class="QLabel" name="lX1_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lX2_2">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lDelta_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCalculer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBQuitter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEquation">
    <property name="title">
     <string>Equation </string>
    </property>
   </widget>
   <addaction name="menuEquation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
